frame/staking/src/lib.rs

```diff
@@ -1835,3 +2309,56 @@ impl<T, Reporter, Offender, R, O> ReportOffence<Reporter, Offender, O>
 		}
 	}
 }
+
+#[allow(deprecated)]
+impl<T: Trait> frame_support::unsigned::ValidateUnsigned for Module<T> {
+	type Call = Call<T>;
+	fn validate_unsigned(call: &Self::Call) -> TransactionValidity {
+		if let Call::submit_election_solution_unsigned(
+			winners,
+			compact,
+			validator_index,
+			signature,
+		) = call {
+			if let Some(queued_elected) = Self::queued_elected() {
+				if queued_elected.compute == ElectionCompute::Authority {
+					debug::native::debug!(
+						target: "staking",
+						"rejecting unsigned transaction because we already have one from an authority"
+					);
+					return InvalidTransaction::Future.into();
```

Comment:
In what way it will become valid in the future? I thought this is only valid for a single session?